gulpfile.js

```diff
@@ -49,7 +70,12 @@ gulp.task('resources', () => {
   ]);
 });
 
-gulp.task('package-resources', ['default'], () => {
+gulp.task('package-init', () => {
+  mkdirp.sync('package');
```

Comment:
Yeah, see previous comment about wanting to avoid a dependency if necessary. But yeah, I'll swap out `mkdirp` for `vinyl-xtreme-shenanigans`